src/modules/private-share-folder/private-sharing.controller.ts

```diff
@@ -239,4 +245,65 @@ export class PrivateSharingController {
         ),
     };
   }
+
+  @Delete('stop/folder-id/:folderId')
```

Comment:
You are deleting a resource here, why not `DELETE /:id` ? As you are deleting a private sharing, then before deleting you match that the user id requesting this OR the `owner_id` OR the `shared_with`